CONTRIBUTING.md

```diff
@@ -55,13 +55,26 @@ pip install -r requirements.txt
 ## Testing
 
 ```shell
-tox -e fmt           # update your code according to linting rules
-tox -e lint          # code style
-tox -e unit          # unit tests
-tox -e integration   # integration tests
-tox                  # runs 'lint' and 'unit' environments
+tox -e fmt                      # update your code according to linting rules
+tox -e lint                     # code style
+tox -e unit                     # unit tests
+tox -e charm-integration        # charm integration tests
+tox -e ha-integration           # high-availability replication integration tests
+tox -e relation-integration     # relation integration tests (legacy and new relations)
```

Comment:
given your other PR, is it worth including a line for `tox -e integration # runs every integration test`?

Reply:
yup!